nifi-nar-bundles/nifi-standard-services/nifi-oauth2-provider-bundle/nifi-oauth2-provider-service/src/main/java/org/apache/nifi/oauth2/StandardOauth2AccessTokenProvider.java

```diff
@@ -338,4 +342,20 @@ private boolean isRefreshRequired() {
 
         return Instant.now().isAfter(expirationRefreshTime);
     }
+
+    @Override
+    public List<ConfigVerificationResult> verify(ConfigurationContext context, ComponentLog verificationLogger, Map<String, String> variables) {
+        ConfigVerificationResult.Builder builder = new ConfigVerificationResult.Builder()
+                .verificationStepName("Can acquire token");
+
+        try {
+            getAccessDetails();
+            builder.outcome(ConfigVerificationResult.Outcome.SUCCESSFUL);
+        } catch (Exception ex) {
```

Comment:
I'm not familiar with what Exceptions might be thrown here, would the message always be appropriate (on its own)?

Would it be sensible to also log the exception, e.g. as a warning with a stacktrace?

Reply:
<img width="757" alt="image" src="https://user-images.githubusercontent.com/108184/201231637-d1e1cbb8-3c5f-4a35-96ec-c7a42fd6217b.png">